ccm_web/server/src/api/dtos/api.section.ids.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { ArrayMaxSize, ArrayUnique, IsArray, IsInt, IsNotEmpty } from 'class-validator'
+import { ApiProperty } from '@nestjs/swagger'
+
+export class SectionIdsDto {
+  @ApiProperty({ type: [Number] })
+  @IsArray()
+  @ArrayMaxSize(250)
+  @ArrayUnique()
+  @IsInt({ each: true })
+  @IsNotEmpty({ each: true })
+  sectionIds: number[]
+
+  constructor (sectionIds: number[]) {
+    this.sectionIds = sectionIds
+  }
+}
```

Comment:
Not sure about the `250` limit here, but it seems responsible to put --some-- limit on this.

Reply:
The use case don't call for bulk create that many section at-least based on todays Canvas usage by Admin staff . 60 is the limit we set and the Backend handles it. But 250 limit seem ok and code can handle it with async calls

Reply:
This would be for merging and un-merging sections. I think I will leave it as is for now, but we can modify it later as needed.

There may be some decorators I'm using here that we should add to the `CreationSectionsDto`, like the `ArrayUnique` and `ArrayMaxSize`. Separate issue though.

Reply:
>This would be for merging and un-merging sections. I think I will leave it as is for now, but we can modify it later as needed.

Sorry I missed noticing that. In that case LIMIT is OK.

May be this dto can have a name like `api.merge.unmerge.section.id's.dto.ts`?

Reply:
I prefer to leave it as general purpose/general naming, since there's nothing about it that is limited to those use cases. If you want me to add a comment indicating where it is used, I can do that.